.github/workflows/release.yml

```diff
@@ -4,6 +4,9 @@ on:
   release:
     types: [published]
 
+  push:
+    branches:
+      - "features/**" # TEMP
```

Comment:
please don't change this - these versions should not be going to PyPI even as a temp thing; github repos can be installed directly into python environments